modules/tracking/test/test_trackers.cpp

```diff
@@ -567,6 +567,22 @@ TEST_P(DistanceAndOverlap, Scaled_Data_CSRT)
   test.run();
 }
 
+TEST(GOTURN, memory_usage)
+{
+  cv::Rect2d roi(145, 70, 85, 85);
+  cv::Mat frame;
+  cv::Ptr<cv::Tracker> tracker = cv::TrackerGOTURN::create();
```

Comment:
Modify `TrackerGOTURN::Params` so it can accept paths to `.caffemodel` and `.prototxt`

Reply:
What if we will change ```gtrTracker.cpp``` like this:
```cpp
bool TrackerGOTURNImpl::initImpl(const Mat& image, const Rect2d& boundingBox)
{
    //Make a simple model from frame and bounding box
    model = Ptr<TrackerGOTURNModel>(new TrackerGOTURNModel(params));
    ((TrackerGOTURNModel*)static_cast<TrackerModel*>(model))->setImage(image);
    ((TrackerGOTURNModel*)static_cast<TrackerModel*>(model))->setBoudingBox(boundingBox);

    //Load GOTURN architecture from *.prototxt and pretrained weights from *.caffemodel
    String modelTxt = cvtest::TS::ptr()->get_data_path() + "/dnn/gsoc2016-goturn/goturn.prototxt";
    String modelBin = cvtest::TS::ptr()->get_data_path() + "/dnn/gsoc2016-goturn/goturn.caffemodel";
    net = dnn::readNetFromCaffe(modelTxt, modelBin);
    return true;
}
```
using variable:
```bash
export OPENCV_TEST_DATA_PATH=/home/ilyaelizarov/opencv_extra/testdata
```
?

Reply:
This would not work ("TS::" is for tests only).

It is better to add extra parameters (into .create() or dedicated .set/.get calls) to specify location of model files with defaults "goturn.caffemodel" / "goturn.prototxt" (from the current directory).

Reply:
OK, I will try this. Thank you for your explanation.